### ISSUE TITLE
Python Port

### ISSUE THREAD

**Issue by tatobari** [mvines/sainsmart-relay16#5]
I've ported these code to Python to be used with Raspberry Pi using "pyusb". @mvines maybe you'd like to add it to these repository. I've seen many people looking for a python version of these code.

**Comment by mvines** [comment]
Interesting, is python code somewhere on github?  I'd be happy to link to it from this project

**Comment by tatobari** [comment]
I should probably tell you that I'm just a web developer currently trying
to learn some Raspberry/Arduino stuff. The code might no be that tidy.

I've just created a repo so you can check the file but I strongly think the
correct thing to do here is to put it in some repo of yours since this is a
port-copy of your code.

The URL is https://github.com/tatobari/hidrelaypy

Let me know and I'll proceed as you say.
On Jun 5, 2016 17:42, "Michael Vines" notifications@github.com wrote:

Interesting, is python code somewhere on github? I'd be happy to link to it
from this project

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
https://github.com/mvines/relay/issues/5#issuecomment-223836371, or mute
the thread
https://github.com/notifications/unsubscribe/AGrZ2SZvad4Mlp2YD-2Nx0IuNikuZkIHks5qIzSagaJpZM4IuZru
.

**Comment by mvines** [comment]
Thanks for sharing.  I've added a link to your repo from the README here (91d47cca3ab72fcdb93b9ab7c45ebddc7e4d632a).  I don't think the python port belongs in this project because:
1. This project is a stand-alone npm-compatible node module.   Adding python into it confuses it's purpose
2. I don't want to support a python port personally, so the code would probably just rot if it was merged here.
3. Looks like you've chosen the GPL as the license for your port.  Totally fine and your choice of course, however I don't want GPL code in this project.
